docs/readme.md

```diff
@@ -10,29 +10,35 @@ assumes you've installed Lighthouse as a dependency (`yarn add --dev lighthouse`
 ```javascript
 const lighthouse = require('lighthouse');
 const chromeLauncher = require('chrome-launcher');
+const fs = require('fs');
```

Comment:
I was just looking at this myself when I updated for the run multiple times docs, can we do async/await :D

WDYT?

```js
const fs = require('fs');
const lighthouse = require('lighthouse');
const chromeLauncher = require('chrome-launcher');
const log = require('lighthouse-logger');

(async () => {
  log.setLevel('info');
  const chrome = await chromeLauncher.launch({chromeFlags: ['--headless']});
  const options = {output: 'html', onlyCategories: ['performance'], port: chrome.port};
  const runnerResult = await lighthouse('https://example.com', options);

  // `.report` is the HTML report as a string
  const reportHtml = runnerResult.report;
  fs.writeFileSync('lhreport.html', reportHtml);

  // `.lhr` is the Lighthouse Result as a JS object
  console.log('Report is done for', lhr.finalUrl);
  console.log('Performance score was', lhr.categories.performance.score * 100);

  await chrome.kill();  
})();
```

Reply:
i'm sold.